grafana/perf/regression-summary.json

```diff
@@ -70,20 +71,348 @@
+                  "text": "4.9.3-2"
+                },
+                "4947": {
+                  "index": 9,
+                  "text": "4.9.4-7"
+                },
+                "4955": {
+                  "index": 10,
+                  "text": "4.9.5-5"
+                },
+                "4961": {
+                  "index": 11,
+                  "text": "4.9.6-1"
+                },
+                "4972": {
+                  "index": 12,
+                  "text": "4.9.7-2"
+                },
+                "41002": {
+                  "index": 30,
+                  "text": "4.10.0-rc.2"
+                },
+                "41003": {
+                  "index": 31,
+                  "text": "4.10.0-rc.3"
+                },
+                "41007": {
+                  "index": 32,
+                  "text": "4.10.0-rc.7"
+                },
+                "41010": {
+                  "index": 39,
+                  "text": "4.10.10"
+                },
+                "41011": {
+                  "index": 40,
+                  "text": "4.10.11"
+                },
+                "41012": {
+                  "index": 41,
+                  "text": "4.10.12"
+                },
+                "41013": {
+                  "index": 42,
+                  "text": "4.10.13"
+                },
+                "41014": {
+                  "index": 43,
+                  "text": "4.10.14"
+                },
+                "41015": {
+                  "index": 44,
+                  "text": "4.10.15"
+                },
+                "41016": {
+                  "index": 21,
+                  "text": "4.10.1-6"
+                },
+                "41021": {
+                  "index": 28,
+                  "text": "4.10.2-1"
+                },
+                "41023": {
+                  "index": 29,
+                  "text": "4.10.2-3"
+                },
+                "41054": {
+                  "index": 49,
+                  "text": "4.10.5-4"
+                },
+                "41066": {
+                  "index": 48,
+                  "text": "4.10.6-6"
+                },
+                "41110": {
+                  "index": 60,
+                  "text": "4.11.10"
+                },
+                "41111": {
+                  "index": 61,
+                  "text": "4.11.11"
+                },
+                "41112": {
+                  "index": 62,
+                  "text": "4.11.12"
+                },
+                "41113": {
+                  "index": 63,
+                  "text": "4.11.13"
+                },
+                "41114": {
+                  "index": 64,
+                  "text": "4.11.14"
+                },
+                "41144": {
+                  "index": 80,
+                  "text": "4.11.4-4"
+                },
+                "41206": {
+                  "index": 65,
+                  "text": "4.12.0.6"
```

Comment:
These two are still inconsistent (4.11.4-4 vs. 4.12.0.6).

Reply:
4.11.4-4 is for odf operator and 4.12.0.6 for ocp

Reply:
Got it (although it's very confusing, but the code is correct).  Just one more small one to fix up.